gems/aws-sigv4/sig/request.rbs

```diff
@@ -0,0 +1,24 @@
+module AWS
+  module Sigv4
+    class Request
+      def initialize: (
+          http_method: String,
+          endpoint: URI::HTTP | URI::HTTPS | String,
+          headers?: Hash[String, String],
```

Comment:
Should be these `?headers` and `?body` instead since they're optional arguments?

Reply:
I agree with @alextwoods .

The syntax for optional keyword arguments is like this.

```suggestion
          ?headers: Hash[String, String],
```

https://github.com/aws/aws-sdk-ruby/blob/8c6f65feb0500775b50153e0c24a56381509c873/gems/aws-sigv4/lib/aws-sigv4/request.rb#L11

Reply:
Fixed!
a522cc1b8a06496539730bbfedd6ab3110050792